src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/OffenderAlert.kt

```diff
@@ -71,7 +80,16 @@ class OffenderAlertBuilder(
     alertStatus = status,
     commentText = commentText,
     verifiedFlag = verifiedFlag,
-
-  )
```

Comment:
this apply feels painful when I just want it in the constructor, but can't because it is a one-to-one - not sure if there is a better way